src/kmeans.jl

```diff
@@ -43,18 +44,37 @@ function kmeans(X::Matrix, k::Int;
                 weights=nothing,
                 init=_kmeans_default_init,
                 maxiter::Integer=_kmeans_default_maxiter, 
+                n_init::Integer=_kmeans_default_n_init,
                 tol::Real=_kmeans_default_tol,
                 display::Symbol=_kmeans_default_display)
+				
 
     m, n = size(X)
     (2 <= k < n) || error("k must have 2 <= k < n.")
-    iseeds = initseeds(init, X, k)
-    centers = copyseeds(X, iseeds)
-    kmeans!(X, centers; 
-            weights=weights, 
-            maxiter=maxiter,
-            tol=tol,
-            display=display)
+	n_init > 0 || throw(ArgumentError("n_init must be greater than 0"))
```

Comment:
Everything is still being indented with tabs. You can do a find-and-replace to change everything at once, if you wish; I do it in Vim like
```viml
:%s/\t/    /g
```